### PR TITLE
update Versions table to 2022.4. 

### LINKED ISSUE #310: Update versions table to 2022.4
2022.4 is currently released but is not in the Versions table. This should be updated and then the binary built by manually running the github action. This should be done before any new data is added to the database.

### PR DESCRIPTION
**Pull request by kelle**
Closes #310

### CODE REVIEW

**Comment by kelle** [comment]
Once this is merged, I will trigger a new database build for the website manually.

File: scripts/updates/update_version.py [hunk 1]
```diff
@@ -15,10 +15,10 @@
...
 # Add new version, add new entries as appropriate
 # Note that start_date and end_date are strings of the date in format YYYY-MM-DD
-data = [{'version': '2022.3',
-         'start_date': '2022-10-11',
-         'end_date': '2022-10-13',
-         'description': 'Added VHS 1256b data'}]
+data = [{'version': '2022.4',
+         'start_date': '2022-10-14',
+         'end_date': '2022-10-16',
+         'description': ''}]
```
Comment (dr-rodriguez) [inline scripts/updates/update_version.py:21]: No short description as to what changed in 2022.4?

Comment (kelle) [reply scripts/updates/update_version.py:21]: No. the changes were trivial.

**Review by Will-Cooper — APPROVED** [review]
LGTM

**Review by dr-rodriguez — APPROVED** [review]

**Pull request merged**